src/lib/Scenes/Favorites/Components/Artworks/index.tsx

```diff
@@ -23,7 +51,14 @@ export class SavedWorks extends React.Component<any, any> {
     )
 
     const Content = (
-      <Container>
+      <Container
+        onScroll={({ nativeEvent }) => {
+          if (this.isCloseToBottom(nativeEvent)) {
+            this.loadMore()
```

Comment:
:) One thing i noticed when i was building this out in my other PR is that when it gets close its really sensitive and fires a ton of times; to fix it I wrapped my call in `_.once` which basically only fires 1 time per render cycle. Not sure how much this applies in react-native context tho

Reply:
Oh that's possible, I have a check in `loadMore` so it only sends one request for data at a time

Reply:
Since we have a util for it now, I'll just go ahead and use it here as well